includes/class-boldgrid-inspirations-deploy.php

```diff
@@ -1427,6 +1439,9 @@ public function deploy_page_sets() {
 
 					if ( $term_id ) {
 						wp_set_object_terms( $post_id, $term_id, $tax_data->taxonomy );
+						$theme_has_cph = true;
+						// This will create the correlation between the original post ID and the new post id.
+						$cph_original_ids[ $tax_data->post_id ] = $post_id;
```

Comment:
The taxonomy was written so that we could easily add to it in the future. There's no guarantee that there's a post_id attribute. We may need to check that it exists first.